src/wizard/obtain_source.jl

```diff
@@ -161,7 +161,13 @@ function download_source(state::WizardState)
         source_path = joinpath(state.workspace, basename(url))
 
         if isfile(source_path)
-            name, ext = splitext(basename(source_path))
+            if occursin(".tar.", basename(source_path))
+                #match everything up to but not including ".tar.*" to strip multiple file extensions
+                m = match(r"^.*(?=(\.tar\.([\s\S]*)))", basename(source_path))
+                name, ext = m.match, m.captures[1]
+            else
+                name, ext = splitext(basename(source_path))
+            end
```

Comment:
I'd do something like
```suggestion
            # Try to match everything up to but not including ".tar.*" to strip multiple file extensions
            m = match(r"^.+(?=(\.tar\.([\s\S]+)))", basename(source_path))
            name, ext = if isnothing(m)
                splitext(basename(source_path))
            else
                m.match, m.captures[1]
            end
```
The idea is that you do the match once, if it's successful you use its result, otherwise go with plain `splitext`.  What do you think?